server/resp/src/main/java/org/infinispan/server/resp/ByteBufferUtils.java

```diff
@@ -224,8 +225,14 @@ public static int stringSize(long x) {
          d = 0;
          x = -x;
       }
-      int p = -10;
-      for (int i = 1; i < 10; i++) {
+      long p = -10;
```

Comment:
I tried a test run and I got weird results:
```
1000000000000:    13
10000000000000:    14
100000000000000:    15
1000000000000000:    16
10000000000000000:    17
100000000000000000:    19
1000000000000000000:    19
'''
is the implementation with the div by 10 much slower?

Reply:
Thanks for testing! Going to push a fix.

> is the implementation with the div by 10 much slower?

I think so, but we need to set a benchmark to be sure.